lib/inspec/cli.rb

```diff
@@ -93,6 +93,13 @@ def check(path) # rubocop:disable Metrics/AbcSize
     exit 1 unless result[:summary][:valid]
   end
 
+  desc 'vendor', 'Download all dependencies and generate a lockfile'
+  def vendor(path = nil)
+    profile = Inspec::Profile.for_target('./', opts)
+    lockfile = profile.generate_lockfile(path)
+    File.write('inspec.lock', lockfile.to_yaml)
```

Comment:
alright, but how would that work if I run:
```
$ inspec exec profile1
$ inspec exec profile2
```

Wouldn't that break the user expectation?